### PR TITLE
5025 recommend flight over flightsql

### PR DESCRIPTION
**Pull request by jstirnaman**
Part of #5025 

Recommend using client libraries over using Flight and Flight SQL clients.
Recommend using Flight clients (using IOx-specific Flight) over using Flight SQL.
Organize Flight and Flight SQL clients.
Link to new client libraries.

### CODE REVIEW

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/csharp-flight.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: C# .NET Flight client
+description: The C# .NET Flight client integrates with golang scripts and applications to query data stored in InfluxDB.
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/csharp-flight.md:3]:
```suggestion
description: The C# .NET Flight client integrates with C# .NET scripts and applications to query data stored in InfluxDB.
```

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/go-flight.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Go Flight client
+description: The Go Flight client integrates with golang scripts and applications to query data stored in InfluxDB.
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/go-flight.md:3]: Same comment as above: "golang" -> "Go"

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        '1970-01-01T00:00:00Z'::TIMESTAMP) AS _time,
+      room,
+      selector_max(temp, time)['value'] AS 'max temp',
+      selector_min(temp, time)['value'] AS 'min temp',
+      avg(temp) AS 'average temp'
+    FROM home
+    GROUP BY
+      _time,
+      room
+    ORDER BY room, time"""
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md:33]:
```suggestion
    SELECT DATE_BIN(INTERVAL '2 hours', time),
      room,
      selector_max(temp, time)['value'] AS 'max temp',
      selector_min(temp, time)['value'] AS 'min temp',
      avg(temp) AS 'average temp'
    FROM home
    GROUP BY
      1,
      room
    ORDER BY room, time"""
```

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      '1970-01-01T00:00:00Z'::TIMESTAMP) AS _time,
+    room,
+    selector_max(temp, time)['value'] AS 'max temp',
+    selector_min(temp, time)['value'] AS 'min temp',
+    avg(temp) AS 'average temp'
+  FROM home
+  GROUP BY
+    _time,
+    room
+  ORDER BY room, time"""
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md:120]:
```suggestion
    SELECT DATE_BIN(INTERVAL '2 hours', time),
      room,
      selector_max(temp, time)['value'] AS 'max temp',
      selector_min(temp, time)['value'] AS 'min temp',
      avg(temp) AS 'average temp'
    FROM home
    GROUP BY
      1,
      room
    ORDER BY room, time"""
```

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+[Apache Arrow Python bindings](https://arrow.apache.org/docs/python/index.html) integrate with Python scripts and applications to query data stored in InfluxDB.
+
+The following examples show how to use the `pyarrow.flight` and `pandas` Python modules to query and format data stored in an {{% cloud-name %}} database:
+
+{{% code-tabs-wrapper %}}
+{{% code-tabs %}}
+[InfluxQL](#influxql-python)
+[SQL](#sql-python)
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md:57]: For v3, I believe we are treating SQL as the primary query language, so SQL should probably be the first tab (default content).

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+[Apache Arrow Python bindings](https://arrow.apache.org/docs/python/index.html) integrate with Python scripts and applications to query data stored in InfluxDB.
+
+The following examples show how to use the `pyarrow.flight` and `pandas` Python modules to query and format data stored in an {{% cloud-name %}} bucket:
+
+{{% code-tabs-wrapper %}}
+{{% code-tabs %}}
+[InfluxQL](#influxql-python)
+[SQL](#sql-python)
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/python-flight.md:57]: Same comment as above about SQL being first.

**Review by sanderson — CHANGES_REQUESTED** [review]
Looks really good. Just a few suggestions / requested changes.

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+        '1970-01-01T00:00:00Z'::TIMESTAMP) AS _time,
+      room,
+      selector_max(temp, time)['value'] AS 'max temp',
+      selector_min(temp, time)['value'] AS 'min temp',
+      avg(temp) AS 'average temp'
+    FROM home
+    GROUP BY
+      _time,
+      room
+    ORDER BY room, time"""
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md:33]:
```suggestion
    ORDER BY room, _time"""
```

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md (hunk 5 above)
Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md:57]: OK, thanks. I thought about that and then decided alphabetical.

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+  SELECT DATE_BIN(INTERVAL '2 hours', time),
+    room,
+    selector_max(temp, time)['value'] AS 'max temp',
+    selector_min(temp, time)['value'] AS 'min temp',
+    avg(temp) AS 'average temp'
+  FROM home
+  GROUP BY
+    1,
+    room
+  ORDER BY room, time"""
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md:118]: @sanderson Doesn't work with `time` in `ORDER BY`, though; nor did my original. Still need to use an alias for the DATE_BIN calculation.

Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md:118]: Oh, sorry, you should be able to use `ORDER BY room, 1`.

Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/python-flight.md:118]:
```suggestion
  ORDER BY room, 1"""
```

File: content/influxdb/cloud-dedicated/reference/client-libraries/flight/go-flight.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+title: Go Flight client
+description: The Go Flight client integrates with Go scripts and applications to query data stored in InfluxDB.
+menu:
+  influxdb_cloud_dedicated:
+    name: Go
+    parent: Arrow Flight clients
+    params:
+      url: https://pkg.go.dev/github.com/apache/arrow/go/v12/arrow/flight#Client
+    identifier: go-flight-client
+influxdb/cloud-dedicated/tags: [Golang, gRPC, SQL, Flight SQL, client libraries]
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/flight/go-flight.md:11]:
```suggestion
influxdb/cloud-dedicated/tags: [Go, gRPC, SQL, Flight SQL, client libraries]
```

File: content/influxdb/cloud-serverless/query-data/influxql/execute-queries/python.md [hunk 10]
```diff
@@ -38,7 +38,7 @@ list_code_example: |
 ---
 
 Use the `influxdb3-python` client library to query data stored in InfluxDB with InfluxQL.
-The `influxdb3-client` uses Flight SQL to query data from InfluxDB and return
+The `influxdb3-client` uses InfluxDB v3's Flight RPC protocol to query data from InfluxDB and return
 results in Apache Arrow format.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/query-data/influxql/execute-queries/python.md:42]:
```suggestion
results in Apache Arrow Table format.
```

File: content/influxdb/cloud-serverless/reference/client-libraries/flight/go-flight.md (hunk 2 above)
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/reference/client-libraries/flight/go-flight.md:3]:
```suggestion
description: The Go Flight client integrates with Go scripts and applications to query data stored in InfluxDB.
```

**Pull request merged**